.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,60 @@
+name: Feature request
```

Comment:
Wrong name

Reply:
Hi @blackfalcon,

I changed the name for this. Thank you.